src/components/Search.scss

```
@@ -37,13 +67,19 @@
     width: 80%;
     max-width: none;
     padding: 8px;
+    padding-left: 30px;
 
     @include media($large-screen) {
       width: 240px;
       max-width: inherit;
     }
   }
 
+  .va-api-search-icon {
+    position: relative;
+    left: 25px;
+    color: $color-white;
```

```suggestion
    color: $color-white;
```

in the design, this is some shade of gray: #C3C8CF. i see where you got it from design.va.gov, but what they're really doing is using white + opacity to get this color: `rgba(255, 255, 255, 0.75)`. we can do that or use the hex value, either way.